components/core/Button.tsx

```diff
@@ -1,22 +1,66 @@
 import { chakra, Button as ChakraButton } from "@chakra-ui/react";
+import theme from "@/src/theme";
+
 
 export const Button = chakra(ChakraButton, {
   baseStyle: {
-    color: "red",
+    font: "Inter",
+    fontSize: "14px",
+    height: "auto",
+    borderRadius: "4px",
+    padding: "16px 24px"
   },
 });
 
 export const RedButton = chakra(ChakraButton, {
```

Comment:
I think we can remove the red button and replace it in the sign up/ login with the black one

Reply:
I think I will fix this in my next PR when I'm adjusting UI for login/signup stuff